sdk/synapse/azure-synapse-artifacts/azure/synapse/artifacts/_artifacts_client.py

```diff
@@ -46,11 +48,15 @@
     from azure.core.credentials import TokenCredential
 
 
-class ArtifactsClient:  # pylint: disable=client-accepts-api-version-keyword,too-many-instance-attributes
-    """ArtifactsClient.
+class ArtifactsClient(
+    ArtifactsClientOperationsMixin
+):  # pylint: disable=client-accepts-api-version-keyword,too-many-instance-attributes
+    """SyMS API Service Client.
```

Comment:
I think maybe Syms API should change its opreationId to format {operationGroup}_{opeartionName} to keep the same behavior as all the other APIs?
For example: Syms_ExecuteChange
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/synapse/data-plane/Microsoft.Synapse/stable/2021-04-01/symsAPI.json#LL30C9-L30C40

@msyyc @piyushdubey Does that make sense to you?

Reply:
Agree that it is better to follow format `{operationGroup}_{opeartionName}` like other API.

Reply:
@wonner - This [PR](https://github.com/Azure/azure-rest-api-specs/pull/24516) addresses your concern about operationIds

Reply:
after https://github.com/Azure/azure-rest-api-specs/pull/24908, we need regenerate the code.